rosidl_generator_cpp/CMakeLists.txt

```diff
@@ -21,11 +23,14 @@ if(BUILD_TESTING)
 
     "msg/Empty.msg"
 
+    "msg/MyVector3.msg"
```

Comment:
Yeah, this is needed because I wanted to make sure we tested a "complex" type with some of the fields set to a default value, and some unset.  I didn't find any other messages that had this particular semantic in `rosidl_generator_cpp/msg`.  I agree with you that the name would be better as something else; I'll take your suggestion and change the name to `FieldsWithSameTypeSomeDefaults`.  As far as keeping the testing closer to the other generators, I would prefer to add additional tests to the other generators (including adding this message) in another PR.  Does that seem reasonable to you?

Reply:
Yeah totally, I meant having a message definition we could copy to the other generators packages for testing. It makes total sense not to touch the other generators in this PR but in follow-up PRs when we add the initialization features to them :+1: .